src/components/AdvocatesTable/columns.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import { ColumnDef } from "@tanstack/react-table";
```

Comment:
I've liked this library better than others I've used. The interface is pretty simple but gives you flexibility to format columns as you like. See `specialties` and `phoneNumber` below.